python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -18,15 +18,21 @@
 SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
 SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
 
+is_runtime_release = Variable.get("IS_RUNTIME_RELEASE", default_var="False")
+
 
 def get_report(dag_run_ids: List[str], **context: Any) -> None:  # noqa: C901
     """Fetch dags run details and generate report"""
+
     with create_session() as session:
         last_dags_runs: List[DagRun] = session.query(DagRun).filter(DagRun.run_id.in_(dag_run_ids)).all()
         message_list: List[str] = []
 
         airflow_version = context["ti"].xcom_pull(task_ids="get_airflow_version")
-        airflow_version_message = f"Airflow version for the below astro-sdk run is `{airflow_version}` \n\n"
+        if is_runtime_release == "TRUE":
```

Comment:
This seems suboptimal, why not using something like `getboolean`?
https://github.com/apache/airflow/blob/54f1fb0574a6ecf8f415bbf6da1aaf6f1999bb29/airflow/configuration.py#L765